doc/api/permissions.md

```diff
@@ -550,6 +550,10 @@ There are constraints you need to know before using this system:
 * Relative paths are not supported through the CLI (`--allow-fs-*`).
 * The model does not inherit to a child node process.
 * The model does not inherit to a worker thread.
+* Symbolic links will be followed even to locations outside of the set of paths
+  that access has been granted to. In particular, the mere existence of a single
+  relative symbolic links within any accessible directory may allow an attacker
+  to access arbitrary files and directories.
```

Comment:
I wonder if we should mention this sentence... That sounds like: "Don't use the permission model at all". Wouldn't the first sentence be enough to clarify the limitation?

Reply:
I think it's fair to point out, as most folks won't necessarily have in mind that aspect of symlinks.

Reply:
> Wouldn't the first sentence be enough to clarify the limitation?

I have requested to disclose https://hackerone.com/reports/1961655 (as the reporter), and once another @nodejs/tsc member approves disclosure (which I unfortunately cannot do as the reporter), that discussion should demonstrate that the first sentence is, in fact, not enough.